### ISSUE TITLE
Training code?

### ISSUE THREAD

**Issue by escorciav** [epic-kitchens/action-models#10]
Hi @willprice 

thanks for providing this. I can't find the training code (loop). Just to confirm, is it out of the scope of the repo?

I guess you used the original author's code. Do you happen to have those public?
I found your tsm-fork, but not the code for epic-kitchen 🙂

Thanks!
Victor

**Comment by willprice** [comment]
Hi @escorciav,
This repo only covers model definitions and trained weights. The training loops are only slight variants on those in the TSN repo (logging top-1/5 accuracy for verbs/nouns/action). I don't have any plans to prepare the training code for release.

**Comment by escorciav** [comment]
Gotcha, thanks!
